cnxepub/adapters.py

```diff
@@ -158,6 +158,7 @@ def _make_package(binder):
                 is_navigation=True, properties=['nav'])
     items.append(item)
     resources = {}
+    page_map = {doc.id: doc.ident_hash for doc in flatten_to_documents(binder)}
```

Comment:
Nice! Could there be a comment that says what this was for (getting the `@{ver}`)?